proto/internal/temporal/server/api/persistence/v1/task_queues.proto

```diff
@@ -71,11 +71,56 @@ message CompatibleVersionSet {
     temporal.server.api.clock.v1.HybridLogicalClock became_default_timestamp = 4;
 }
 
+message RampByPercentage {
+    // Acceptable range is [0,100].
+    float ramp_percentage = 1;
```

Comment:
not to open a can of worms here, but do we really want floating point values? and if we do, do we want it to be a percentage out of 100, or a fraction out of 1? a fraction of 1 is a little cleaner in some ways, but a percent is more familiar. (I know this would imply a change to api too)

Reply:
I think there would be users for whom 1% is too much for the very first step. I'd be fine with a fraction of 1 but then we need to come up with a better term in the names instead of 'percentage'.

Reply:
I didn't mean integer percentage, just fixed point. E.g. number out of 1 million or 1 billion